filenames.gni

```diff
@@ -665,8 +665,8 @@ filenames = {
     "chromium_src/chrome/browser/process_singleton.h",
     "chromium_src/chrome/browser/ui/views/frame/global_menu_bar_registrar_x11.cc",
     "chromium_src/chrome/browser/ui/views/frame/global_menu_bar_registrar_x11.h",
-    "chromium_src/chrome/renderer/spellchecker/spellcheck_worditerator.cc",
-    "chromium_src/chrome/renderer/spellchecker/spellcheck_worditerator.h",
+    "//components/spellcheck/renderer/spellcheck_worditerator.cc",
+    "//components/spellcheck/renderer/spellcheck_worditerator.h",
```

Comment:
Can you move this to sources list in `BUILD.gn` https://github.com/electron/electron/blob/master/BUILD.gn#L297 , this file only contains sources under //electron.

Reply:
@deepak1556 done